lib/Raven/SerializerInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+
+namespace Raven;
+
+interface SerializerInterface
+{
+    /**
+     * Serialize an object (recursively) into something safe for data
+     * sanitization and encoding.
+     *
+     * @param mixed $value
+     * @param int   $max_depth
+     * @param int   $_depth
+     * @return string|bool|double|int|null|object|array
+     */
+    public function serialize($value, $max_depth = 3, $_depth = 0);
```

Comment:
What about changing the signature of this method to `serialize($value, array $context = [])?` This way, it's more standardized according to other serializer libraries and mostly important it allows us to pass additional options in the future without breaking compatibility by adding a new method to the interface